packages/runtime-dom/src/index.ts

```diff
@@ -63,7 +65,11 @@ export const createApp = ((...args) => {
     }
     // clear content before mounting
     container.innerHTML = ''
-    return mount(container)
+    const proxy = mount(container)
+    if (container.hasAttribute(ATTR_CLOAK)) {
+      container.removeAttribute(ATTR_CLOAK)
```

Comment:
We can just call remove here. No need to check it first. With that, it's also unnecessary to define a variable for it.